src/test/java/org/jabref/logic/importer/fileformat/PdfContentImporterTest.java

```diff
@@ -121,6 +126,29 @@ British Journal of Nutrition (2008), 99, 1–11 doi: 10.1017/S0007114507795296
                 British Journal of Nutrition
                 https://doi.org/10.1017/S0007114507795296 Published online by Cambridge University Press""";
 
-        assertEquals(Optional.of(entry), importer.getEntryFromPDFContent(firstPageContent, "\n"));
+        assertEquals(Optional.of(entry), importer.getEntryFromPDFContent(firstPageContent, "\n", ""));
+    }
+
+    @ParameterizedTest
+    @MethodSource("providePdfData")
+    void pdfTitleExtraction(String filePath, String expectedTitle) throws Exception {
```

Comment:
Follow-up pull request: swap the arguments.

at `assert` in JUnit, first the expected value appears, then the input data. - This should also be done when wrapping inside `@ParameterizedTests`